src/Bokeh.jl

```diff
@@ -1,12 +1,15 @@
+VERSION >= v"0.4.0-dev+6641" && __precompile__()
```

Comment:
This marks the commit that brought precompile in. It will include all RCs

Would we want to use anything else?